### ISSUE TITLE
[Crash] Lanterns Belong on Walls

### ISSUE THREAD

**Issue by SylisMC** [WolfieWaffle/Hardcore-Torches-Fabric#11]
**Game Version**
1.19.2

[Hardcore Torches - Github thread](https://github.com/WolfieWaffle/Hardcore-Torches-Fabric/issues/11)
[Lanterns Belong On Walls - Github thread](https://github.com/ThePoultryMan/Lanterns-Belong-On-Walls/issues/13)

**Mod List**
fabric-api-0.71.0+1.19.2.jar
hardcore_torches_fabric-1.19-1.6.1a.jar 
lanterns-belong-on-walls-1.4.0-1.19.jar

Game crashes when trying to create a new world

[latest.log](https://github.com/WolfieWaffle/Hardcore-Torches-Fabric/files/10347765/latest.log)

```
The game crashed whilst mouseclick event handler
Error: java.lang.IllegalArgumentException: Cannot get property class_2753{name=facing, clazz=class net.minecraft.class_2350, values=[north, east, south, west, up, down]} as it does not exist in Block{hardcore_torches:lit_lantern}
```

Basically, its caused by [lanterns belong on walls](https://www.curseforge.com/minecraft/mc-mods/lanterns-bow) trying to determine what to do when a player tries to place a [Hardcore Torches](https://www.curseforge.com/minecraft/mc-mods/hardcore-torches) lantern on a wall. 

I think one of you just needs a mouseclick event to stop the crash and a texture so it doesn't look stupid

**Comment by WolfieWaffle** [comment]
I will ask the modding discord about this one, it seems simple enough to at least remove the crash. making compatibility may be more difficult but the crash should be easy to fix.
